fossunited/api/schedule.py

```diff
@@ -94,3 +96,30 @@ def get_schedule_by_hall(schedule: list) -> dict:
         _schedule[hall].append(session)
 
     return _schedule
+
+
+@frappe.whitelist(allow_guest=True)
+def get_schedule_session_ics(schedule: dict):
+    """
+    Download the ics file for the schedule session
+
+    Args:
+        schedule (dict): Schedule session data
+    """
+
+    cal = Calendar()
+
+    timezone = pytz.timezone("Asia/Kolkata")
+
+    event_location = frappe.db.get_value("FOSS Chapter Event", schedule.get("parent"), "event_location")
+    event = Event()
+    event.name = schedule.get("title")
```

Comment:
I would suggest to add the event name in the Title too - it currently says about just session title, which can be confusing at some points too. 
<img width="1042" alt="Screenshot 2024-08-29 at 9 37 23 PM" src="https://github.com/user-attachments/assets/5141cb6c-1ce9-436e-871e-80ca07396487">